flocker/docs/_version.py

```diff
@@ -135,14 +137,23 @@ def is_release(version):
 
 def is_weekly_release(version):
     """
-    Return whether the version corresponds to a weekly release.
-
     :param bytes version: A version of flocker.
-
-    :return bool: Wether the version is a weekly release.
+    :return bool: Whether the version is a weekly release.
     """
     parsed_version = parse_version(version)
     return (parsed_version.weekly_release is not None
             and parsed_version.commit_count is None
             and parsed_version.pre_release is None
             and parsed_version.dirty is None)
+
+
+def is_pre_release(version):
+    """
+    :param bytes version: A version of flocker.
+    :return bool: Whether the version is a pre-release.
+    """
+    parsed_version = parse_version(version)
```

Comment:
Docstrings for all 3 `is_*release` functions should include an initial description, as well as the parameters.